app/controllers/journeys/SectionCompletedStateController.scala

```diff
@@ -35,7 +35,7 @@ import scala.concurrent.{ExecutionContext, Future}
 
 class SectionCompletedStateController @Inject()(override val messagesApi: MessagesApi,
                                                 selfEmploymentConnector: SelfEmploymentConnector,
-                                                navigator: Navigator,
+                                                navigator: IncomeNavigator,
```

Comment:
Does it make sense for the section completed controller to use the income navigator? Would it be better to have a generic navigator for pages not tied to a specific journey (although there is just the one page at the moment)?